.github/workflows/webpack.yml

```
@@ -0,0 +1,28 @@
+name: NodeJS with Webpack
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)